### PR TITLE
chore(deps): updated graphql-js version to support 0.10.0

### PR DESCRIPTION
**Pull request by dotansimha**
Also updated `subscriptions-transport-ws` version to the latest.

TODO:

- [x] Update CHANGELOG.md with your change (include reference to issue & this PR)
- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests and linter rules pass

### CODE REVIEW

File: packages/graphql-server-module-graphiql/src/renderGraphiQL.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export function renderGraphiQL(data: GraphiQLData): string {
   <script src="//cdn.jsdelivr.net/react/15.0.0/react-dom.min.js"></script>
   <script src="//cdn.jsdelivr.net/graphiql/${GRAPHIQL_VERSION}/graphiql.min.js"></script>
   ${usingSubscriptions ?
-    '<script src="//unpkg.com/subscriptions-transport-ws@0.5.4/browser/client.js"></script>' +
+    '<script src="//unpkg.com/subscriptions-transport-ws@0.7.0/browser/client.js"></script>' +
```
Comment (DxCx) [inline packages/graphql-server-module-graphiql/src/renderGraphiQL.ts]: Lets put under const like graphiql version. Also, graphiql version can be updated to 0.10.2

Comment (dotansimha) [inline packages/graphql-server-module-graphiql/src/renderGraphiQL.ts]: Done

**Comment by dotansimha** [comment]
@DxCx 

1 - Done
2 - Why? It thought we want to keep it with the same for query and mutations, but use the ws only for subscriptions (in graphiql)

**Comment by mxstbr** [comment]
Yess, this is great! :tada: Can't wait for this to land to try the new subscriptions stuff :blush:

**Comment by DxCx** [comment]
well, you rushed to merge it but this is incomplete:
https://github.com/apollographql/graphql-server/blob/master/package.json#L34

packages will still use 0.9.2 for testing...

File: packages/graphql-server-module-operation-store/package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/apollostack/graphql-server#readme",
   "peerDependencies": {
-    "graphql": "^0.8.0 || ^0.9.0"
+    "graphql": "^0.8.0 || ^0.9.0 || ^0.10.0"
```
Comment (DxCx) [inline packages/graphql-server-module-operation-store/package.json]: hey @dotansimha you forgot to align with others :)

**Pull request merged**